location/interface.go

```diff
@@ -7,5 +7,11 @@ type Resolver interface {
 
 // Detector allows detecting location by current ip
 type Detector interface {
-	DetectCountry() (string, error)
+	DetectLocation() (Location, error)
+}
+
+// Cache allows caching location
+type Cache interface {
+	Get() (Location, error)
+	RefreshAndGet() (Location, error)
```

Comment:
Most of uses of `RefreshAndGet` seems to be made only for `Refresh` part, result is ignored.
We can simplify this method to assigning single responsibility to it - just `Refresh()`.

Reply:
I agree